README.md

```diff
@@ -6,6 +6,8 @@ Markdown viewer and editor for the [Zooniverse](https://www.zooniverse.org). Req
 
 Available on [npm](http://npmjs.com), include as a dependency using `npm install --save markdownz`
 
+Any default styles can be added to `src/css/markdown-help.css`. Don't forget to add a `require` statement for the css file!
```

Comment:
This isn't quite clear to me if I didn't know what we did. I think there may be two things we'll want to explain, how as a contributor or dev working in the repo we should include styles (currently the first sentence touches on), and how someone using this as a dependency should use the styles (current the second sentence touches on), but I think they could be separate sentences and explained in a little more detail?